src/client/pages/Brokers.tsx

```diff
@@ -40,6 +43,23 @@ const Brokers = () => {
     pollInterval: 20000,
   });
 
+  // various counts from card metric query
+  const counts = useQuery(DASHBOARD_CARD_METRICS_QUERY, {
+    variables: {
+      request: "FetchUnderRep",
+      brokerIds: filter.length > 0 ? filter : null,
+    },
+    pollInterval: 20000,
+  });
+
+  const test = useQuery(BROKER_PAGE_QUERY, {
```

Comment:
This query won't filter correctly by broker, either update the schema or query to look at this at a broker level.